app/src/main/java/com/aacademy/homework/worker/MovieUpdateWorkerRepository.kt

```diff
@@ -0,0 +1,22 @@
+package com.aacademy.homework.worker
+
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequestBuilder
+import com.aacademy.homework.BuildConfig
+import java.util.concurrent.TimeUnit
+
+object MovieUpdateWorkerRepository {
```

Comment:
"it seems like it is a factory for request. Or you can just put it into hilt. But it does not look like Repository" ©Niksl